### PR TITLE
i18n(es): More translated pages

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
More translated pages to spanish:
 - `unknown-csserror.mdx`
 - `unknown-vite-error.mdx`
 - `unknown-markdown-error.mdx`

### CODE REVIEW

File: src/content/docs/es/reference/errors/unknown-csserror.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
+
+title: Unknown CSS Error.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+## ¿Qué salió mal?
+Astro encontró un error desconocido al analizar su CSS. A menudo, esto se debe a un error de sintaxis y el mensaje de error debe contener más información.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/reference/errors/unknown-csserror.mdx:13]:
```suggestion
Astro encontró un error desconocido al analizar el CSS. A menudo, esto se debe a un error de sintaxis y el mensaje de error debe contener más información.
```

File: src/content/docs/es/reference/errors/unknown-markdown-error.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
+
+title: Unknown Markdown Error.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+## ¿Qué salió mal?
+Astro encontró un error desconocido al analizar su Markdown. A menudo, esto se debe a un error de sintaxis y el mensaje de error debe contener más información.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/reference/errors/unknown-markdown-error.mdx:13]:
```suggestion
Astro encontró un error desconocido al analizar el Markdown. A menudo, esto se debe a un error de sintaxis y el mensaje de error debe contener más información.
```

File: src/content/docs/es/reference/errors/unknown-vite-error.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
+
+title: Unknown Vite Error.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+## ¿Qué salió mal?
+Vite encountered an unknown error while rendering your project. We unfortunately do not know what happened (or we would tell you!)
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/reference/errors/unknown-vite-error.mdx:13]:
```suggestion
```

File: src/content/docs/es/reference/errors/unknown-vite-error.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Vite encontró un error desconocido mientras renderizaba su proyecto. Desafortunadamente no sabemos qué pasó (o ¡te lo diríamos!)
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/reference/errors/unknown-vite-error.mdx:14]:
```suggestion
Vite encontró un error desconocido mientras renderizaba tu proyecto. Desafortunadamente no sabemos qué pasó (o ¡te lo diríamos!)
```

File: src/content/docs/es/reference/errors/unknown-vite-error.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Si puedes reproducir de forma confiable este error, nos gustaría que [abrieras un problema](https://astro.build/issues/)
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/reference/errors/unknown-vite-error.mdx:16]:
```suggestion
Si puedes reproducir de forma confiable este error, nos gustaría que [abrieras una issue](https://astro.build/issues/)
```

**Comment by dreyfus92** [comment]
Gracias por el review @kevinzunigacuellar 💚

**Pull request merged**